VTEX - Promotions & Taxes API.json

```diff
@@ -1255,14 +1295,19 @@
                                                 "type": "string",
                                                 "description": "Archived item."
                                             }
+                                        },
+                                        "nominalDiscountType": {
+                                            "type": "string",
+                                            "description": "Controls the behavior of the NominalDiscount effect. This new field only accepts two string values: `item` and `cart`. The `Cart` option keeps the behavior as it currently: the whole order/cart receives a nominal discount that is distributed among the items. The `Item` option, on the other hand, applies the intended nominal discount on every item present on the cart.",
```

Comment:
Usa bullet points pra separar a descrição de cada um:
- **item:** lorem ipsum
- **cart:** lorem ipsum

Pra isso você deve escrever normal em markdown, depois escapar o markdown pra ficar em texto corrido como valor da descrição.

Reply:
Faltou um "is" em  "as it currently is"